### PR TITLE
Max quota follow up 

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8536
- Fixes https://github.com/owncloud/web/issues/8575

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/33448/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33448/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-alice-2023-3-8-11-27-02.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33448/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-brian-2023-3-8-11-27-32.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33448/tracing/alice-can-share-this-weeks-meal-plan-with-all-parents-carol-2023-3-8-11-27-38.zip 


</p></details>
:boom: The e2e-oc10 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/33458/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33458/tracing/admin-user-can-change-personal-quotas-for-users-admin-2023-3-8-12-31-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33458/tracing/admin-user-can-change-personal-quotas-for-users-alice-2023-3-8-12-31-38.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33458/tracing/admin-user-can-change-personal-quotas-for-users-brian-2023-3-8-12-31-40.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33458/tracing/spaces-can-be-managed-in-the-admin-settings-alice-2023-3-8-12-30-16.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/33458/tracing/unstructured-collection-of-testable-space-interactions-alice-2023-3-8-12-38-16.zip 


</p></details>
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

File: packages/web-pkg/src/components/QuotaSelect.vue [hunk 1]
```diff
@@ -120,34 +108,46 @@ export default {
...
-            maxQuota: this.maxQuota * Math.pow(10, -9)
+          error: this.$gettext('Please enter a value equal to or less than %{ maxQuota }', {
+            maxQuota: this.getFormattedFileSize(this.maxQuota)
+          }),
+
+          selectable: false
+        }
+      }
+
+      if (value > 1e15) {
```
Comment (kulmann) [inline packages/web-pkg/src/components/QuotaSelect.vue:138]: Can't you omit this case if you define the `maxQuota` with a fallback to `1e15` if it's undefined?

Comment (kulmann) [reply packages/web-pkg/src/components/QuotaSelect.vue:138]: Eh, it's a prop. So then just define a local variable in the scope of the `createOption` method:
```js
const maxQuota = this.maxQuota || 1e15
```

Comment (JanAckermann) [reply packages/web-pkg/src/components/QuotaSelect.vue:138]: GOOD CATCH THO!

Comment (JanAckermann) [reply packages/web-pkg/src/components/QuotaSelect.vue:138]: But we can overwrite the default value =)

**Review by JammingBen — APPROVED** [review]

**Pull request merged**